lib/relaton_ccsds/data_fetcher.rb

```diff
@@ -39,7 +39,7 @@ def agent
     end
 
     def index
-      @index ||= Relaton::Index.find_or_create "CCSDS", file: "index-v2.yaml"
+      @index ||= Relaton::Index.find_or_create "CCSDS", file: "index-v2.yaml", pubid_class: Pubid::Ccsds::Identifier
```

Comment:
We need to keep generating previous index file version `index-v1.yaml` to support previous relaton-ccsds versions until all users update the gem. Please add another instance variable for previous index format with string IDs.